components/uploader.py

```diff
@@ -1,11 +1,41 @@
+import os
+
 import dash_html_components as html
 import dash_core_components as dcc
 import dash_table
-from utils import dataparser
+import pandas as pd
+
+from utils import dataparser, earthquake_data
+
+
+def get_table(session_id):
+    """Fetch catalog data from cache and return it as a dash DataTable.
+
+    Keywords arguments:
+    session_id -- ID of the current session
+    """
+    eq_data = earthquake_data.get_earthquake_data(session_id)
+
+    if eq_data is not None and eq_data.data.shape != (0, 0):
+        return dash_table.DataTable(
+            data=eq_data.data[:100].to_dict('records'),
+            columns=[
+                {'name': i, 'id': i} for i in eq_data.data.columns
+            ],
+            style_table={
+                'overflow': 'auto',
+                'width': '98%',
+                'margin': '1%'
+            }
```

Comment:
I believe this styling causes the data table to not be completely visible. At least for me, the first half of the year column is missing.